doc/releases/changelog-dev.md

```diff
@@ -133,6 +133,9 @@ array([False, False])
 * The experimental `DefaultQubit2` device now supports computing VJPs and JVPs using the adjoint method.
   [(#4374)](https://github.com/PennyLaneAI/pennylane/pull/4374)
 
+* Updated `default_expand_fn()` to allow `StatePrep` operations to be applied in the middle of a circuit.
```

Comment:
```suggestion
* Updated `default_expand_fn()` to allow `StatePrep` operations to exist in the middle of a provided circuit.
```

I'm just wondering about our phrasing to describe this change. Maybe we could also say `decompose `StatePrep` operations present in the middle of the circuit`?

Reply:
Yes that's a better description of what we are allowing